frontend-react/src/TelemetryService.test.ts

```diff
@@ -0,0 +1,325 @@
+                    const message = "hello there";
+                    const data = { a: 1 };
+
+                    console.log(message, data);
+
+                    expect(appInsights?.trackEvent).toBeCalledWith({
+                        name: `LOG - ${message}`,
+                        properties: {
+                            severityLevel: SeverityLevel.Information,
+                            message,
+                            additionalInformation: JSON.stringify([data]),
+                        },
+                    });
+                });
+            });
+        });
+
+        describe("when calling console.warn", () => {
+            describe("when not passing in more arguments", () => {
+                test("calls trackEvent with the correct message and severity level", () => {
+                    const message = "hello there";
+
+                    console.warn(message);
+
+                    expect(appInsights?.trackEvent).toBeCalledWith({
+                        name: `WARN - ${message}`,
+                        properties: {
+                            severityLevel: SeverityLevel.Warning,
+                            message,
+                            additionalInformation: undefined,
+                        },
+                    });
+                });
+            });
+
+            describe("when passing in more arguments", () => {
+                test("calls trackException with the correct message, severity level, and stringified additional details", () => {
+                    const message = "hello there";
+                    const data = { a: 1 };
+
+                    console.warn(message, data);
+
+                    expect(appInsights?.trackEvent).toBeCalledWith({
+                        name: `WARN - ${message}`,
+                        properties: {
+                            severityLevel: SeverityLevel.Warning,
+                            message,
+                            additionalInformation: JSON.stringify([data]),
+                        },
+                    });
+                });
+            });
+        });
+
+        describe("when calling console.error", () => {
+            describe("when passing in an Error as the first parameter", () => {
+                test("calls trackException with the correct Error and severity level", () => {
+                    const error = new Error("test error");
+
+                    console.error(error);
+
+                    expect(appInsights?.trackException).toBeCalledWith({
+                        exception: error,
+                        id: error.message,
+                        severityLevel: SeverityLevel.Error,
+                        properties: {
+                            additionalInformation: undefined,
+                        },
+                    });
+                });
+            });
+
+            describe("when passing in a String as the first parameter", () => {
```

Comment:
This was the case that was causing the `not_specified` errors to be added, so I'm just casting to an Error instance now.  I'm curious if SimpleReport has encountered this at all since they're allowing `undefined` [here](https://github.com/CDCgov/prime-simplereport/blob/main/frontend/src/app/TelemetryService.ts#L65)!

@kevinhaube @penny-lischer if you get a chance, would you mind checking in on the Azure side to see if no longer getting `not_specified` errors?  I don't have access so the most I can do is verify that we're sending the proper request payload 😬